pkg/vtpm/build.yml

```diff
@@ -1,6 +1,8 @@
 image: eve-vtpm
 org: lfedge
 config:
+  uid: vtpm
```

Comment:
Can't you pass a specific uid to adduser?

Reply:
yes I can.

Reply:
Seems linuxkit only [respects the value if it is a number](https://github.com/linuxkit/linuxkit/blob/4f89f4f67e392ffa8c8bab63dfaf31746e3c11a0/src/cmd/linuxkit/moby/config.go#L708), otherwise a [incrementally chosen UID is assigned](https://github.com/linuxkit/linuxkit/blob/4f89f4f67e392ffa8c8bab63dfaf31746e3c11a0/src/cmd/linuxkit/moby/build/build.go#L188).

Reply:
vtpm is 14th container declared in `rootfs.yml` as result it should get uid 114 (100+14) which matches the run-time uid it see in the container.

Reply:
#3989